### DISCUSSION TITLE
Performance issues with mobile view and lighthouse for empty(Hello World) NextJs App

### DISCUSSION THREAD

**Discussion by abhikumar22** [vercel/next.js#26929] [Help]
### What example does this report relate to?

Performance issues in mobile view with lighthouse

### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

v10.22.0

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

I have Deployed a Newly created NextJs hello world empty application in the vercel and have checked the lighthouse performance.
In Desktop it is good, every parameter (first contentful paint, time to interact etc is less than 0.3s)

but in mobile performance, all parameters are coming close to 1 seconds for dummy project.
Also getting a warning for remove unused javascript.
_Time to Interact is also more than 1.5_

Project Repo link :- https://github.com/abhikumar22/nextjsdemo
Live link :- https://nextjsdemo-olive.vercel.app/

**Mobile Lighthouse Report** 
<img width="847" alt="Screenshot 2021-07-05 at 2 05 46 PM" src="https://user-images.githubusercontent.com/22506234/124444963-bf5b4b00-dd9c-11eb-8f3f-cdbe885e2ef7.png">

**Desktop Lighthouse Report**
<img width="845" alt="Screenshot 2021-07-05 at 2 25 28 PM" src="https://user-images.githubusercontent.com/22506234/124445070-da2dbf80-dd9c-11eb-8ca4-69a89ed2e866.png">

### Expected Behavior

For a empty hello world project, the stats should be lesser as we have not used anything in the project

### To Reproduce

Goto this link :- https://nextjsdemo-olive.vercel.app/

and in lighthouse, perform mobile test

**Comment by timneutkens** [comment]
This is likely related to how lighthouse throttles and reduces cpu speed during the mobile test. Running a single test in lighthouse, especially on your machine locally, is just a reference point of where you can optimize and not a set in stone performance metric.

**Reply by abhikumar22** [reply]
have come across multi discussions/issues where user's are reporting the same regarding this.
i am also facing this

https://github.com/vercel/next.js/discussions/13674
https://github.com/vercel/next.js/discussions/16308
https://github.com/vercel/next.js/issues/17627
https://stackoverflow.com/questions/66830108/why-is-the-performance-score-of-an-empty-default-nextjs-application-so-low
https://github.com/GoogleChrome/lighthouse/issues/10904

some of this are here

**Reply by timneutkens** [reply]
In both screenshots provided the performance score is **100**. Given that the test was local there's other parameters that affect the timings as well:

- Location compared to the edge serving the request (latency and connection jitter of the provider)
- Machine specs
- Browser extensions

As said you'll want to run a lab tests on hardware with the optimal conditions (e.g. webpagetest) to measure the results, and then you have to run it many times to get a representitive result

**Reply by abhikumar22** [reply]
yes performance is 100% but the parameters are different.

check the FCP, TTI

**Reply by timneutkens** [reply]
As said that's expected, when lighthouse runs the test for mobile it throttles your network connection and the CPU is slowed down by noise.

**Reply by abhikumar22** [reply]
yes that i understood but why these stat's are more in mobile view, not in desktop.
There must be some issue here.

if network throttling is the case then desktop should also have the similar results right !!!!!

why only mobile ?

**Reply by timneutkens** [reply]
> if network throttling is the case then desktop should also have the similar results right !!!!!

No because lighthouse only throttles for the mobile report
